beaconChain/constants/constants.ts

```diff
@@ -2,21 +2,46 @@
 // TODO Update TBD
 // TODO Explain what each constant does
 
-export default {
-  BASE_REWARD_QUOTIENT: 32768,
-  CYCLE_LENGTH: 64,
-  DEPOSIT_SIZE: 32,
-  GENESIS_TIME: "TBD",
-  GWEI_PER_ETH: 1000000000,
-  INITIAL_FORK_VERSION: 0,
-  LOGOUT_MESSAGE: "LOGOUT",
-  MAX_VALIDATOR_CHURN_QUOTIENT: 32,
-  MIN_COMMITTEE_SIZE: 128,
-  MIN_ONLINE_DEPOSIT_SIZE: 16,
-  MIN_VALIDATOR_SET_CHANGE_INTERVAL: 256,
-  RANDAO_SLOTS_PER_LAYER: 4096,
-  SHARD_COUNT: 1024,
-  SLOT_DURATION: 16,
-  SQRT_E_DROP_TIME: 65536,
-  WITHDRAWAL_PERIOD: 524288,
-};
+  // Misc
+export const SHARD_COUNT = 2 ** 10; // 1024 shards
+export const TARGET_COMMITTEE_SIZE = 2 ** 8; // 256 validators
+export const MIN_BALANCE = 2 ** 4; // 16 ETH
+export const MAX_BALANCE_CHURN_QUOTIENT = 2 ** 5; // 32
+export const GWEI_PER_ETH = 10 ** 9; // 1B Gwei/ETH
+export const BEACON_CHAIN_SHARD_NUMBER = 2 ** 64 - 1;
+export const BLS_WITHDRAWAL_PREFIX_BYTE = 0x0;
```

Comment:
the spec outlines `0x00`, not sure if this actually causes any issues @ansermino ?